compose/app/build.gradle

```diff
@@ -0,0 +1,62 @@
+plugins {
+    id 'com.android.application'
+    id 'kotlin-android'
+}
+
+android {
+    compileSdk 31
```

Comment:
Shouldn't be same as in others build.gradle files? `AndroidVersions.compileSdkVersion`

Reply:
we are forced to move this into a dedicated compose folder that doesn't use anything from the root due to gradle-plugin/kotlin versions changes needed to run compose. We can't atm update those in the root project due to upstream dependencies :sad_panda: